op-deployer/pkg/deployer/bootstrap/opcm.go

```diff
@@ -28,10 +30,16 @@ import (
 )
 
 type OPCMConfig struct {
-	L1RPCUrl         string
-	PrivateKey       string
-	Logger           log.Logger
-	ArtifactsLocator *opcm.ArtifactsLocator
+	L1RPCUrl                        string
+	PrivateKey                      string
+	Logger                          log.Logger
+	ArtifactsLocator                *opcm.ArtifactsLocator
+	WithdrawalDelaySeconds          uint64
+	MinProposalSizeBytes            uint64
+	ChallengePeriodSeconds          uint64
+	ProofMaturityDelaySeconds       uint64
+	DisputeGameFinalityDelaySeconds uint64
+	MIPSVersion                     uint64
```

Comment:
Instead of listing these fields out individually, could we nest the `SuperchainProofParams` struct ([link](https://github.com/ethereum-optimism/optimism/blob/feat/override-proof-params/op-deployer/pkg/deployer/pipeline/implementations.go#L14-L21))? Would help reduce redundant code